funnel/views/errors.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import unicode_literals
+
+from flask import current_app, redirect, request
+from werkzeug import exceptions
```

Comment:
These two lines are _not_ identical:

```python
import werkzeug.exceptions as exceptions
from werkzeug import exceptions
```

The first is the correct way to do it. The second will only work if the `werkzeug` package has `from . import exceptions`, thereby re-exporting the module as a symbol. It is not safe to assume this unless it is documented.